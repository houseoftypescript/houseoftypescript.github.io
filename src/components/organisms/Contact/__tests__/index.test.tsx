import { render } from '@testing-library/react';
import { Contact } from '../index';

describe('Contact', () => {
  it('renders correctly', () => {
    const button = render(<Contact></Contact>);
    expect(button.container).toMatchSnapshot();
  });
});
