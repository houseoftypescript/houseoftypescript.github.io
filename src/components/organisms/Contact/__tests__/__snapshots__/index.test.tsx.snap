// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact renders correctly 1`] = `
<div>
  <section
    class="py-0 md:py-16 bg-[#F8F0EA] bg-bottom bg-contain bg-no-repeat"
    id="contact"
    style="background-image: url(/img.jpg);"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    >
      <div
        class="bg-white rounded-none md:rounded-2xl"
      >
        <div
          class="px-4 py-8 md:px-8 md:py-16"
        >
          <div
            class="grid grid-cols-1 md:grid-cols-2 items-center gap-8"
          >
            <div
              class="col-span-1"
            >
              <div
                class=""
              >
                <div
                  class="text-center md:text-left max-w-lg mx-auto"
                >
                  <h1
                    class="text-2xl sm:text-4xl mb-4 font-b162 font-bold"
                  />
                  <h2
                    class="text-sm sm:text-lg"
                  />
                </div>
              </div>
            </div>
            <div
              class="col-span-1"
            >
              <div
                class="grid grid-cols-1 md:grid-cols-3 gap-4"
              >
                <div
                  class="col-span-1 md:col-span-2"
                >
                  <input
                    class="w-full px-8 py-4 rounded-md bg-[#EFF3F7] hover:border-emerald-500 active:border-emerald-500 focus:border-emerald-500"
                    id="email"
                    name="email"
                    placeholder="Enter Email Address"
                    type="text"
                  />
                </div>
                <div
                  class="col-span-1"
                >
                  <button
                    class="px-8 py-4 rounded-md bg-emerald-500 hover:bg-gray-900 text-white transition duration-500 w-full"
                    type="button"
                  >
                    Contact
                  </button>
                </div>
                <div
                  class="col-span-1 md:col-span-3"
                >
                  <label
                    class="flex items-center gap-2"
                    for="promote"
                  >
                    <input
                      class="hidden"
                      id="promote"
                      name=""
                      type="checkbox"
                      value=""
                    />
                    <div
                      class="w-5 h-5 rounded border border-2"
                    />
                    <span
                      class="text-sm text-gray-500"
                    >
                      Don't provide any promotional message.
                    </span>
                  </label>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
